### DISCUSSION TITLE
Cannot Continue Poe.com Conversations

### DISCUSSION THREAD

**Discussion by gullpavon** [snowby666/poe-api-wrapper#44] [Q&A]
I can't seem to continue conversations, 

I start by iniating: 

```
bot = "beaver"
message = "what are M1 chips?"

# Create new chat thread
# Streamed example:
client.send_message(bot, message, )

```

and then i try to continue response using the chat code provided from initial message:

```
bot = 'beaver'
message ='summarize the above in bullets'

client.send_message(bot, message, chatCode='1klcvn5ws9ne6en1f2z')
```

But all i get is the original response, going into Poe.com and checking manually i see that the subsequent message was not sent -- any ideas?

**Comment by snowby666** [comment]
This will not send the message
its supposed to be:
```py
for chunk in client.send_message(bot, message, chatCode='1klcvn5ws9ne6en1f2z'):
     pass
```

**Reply by gullpavon** [reply]
Thanks @snowby666 , tried this but it did not send my subsequent message ... 

To clarify, I attempted to bypass using a for loop to iterate over "chunks" because when I tested this API and printed each "chunk" within the loop, it appeared to be iterating over each individual character in the response string rather than over separate response messages or data objects. Checking Poe.com manually, I don't see the second prompt being sent

So concretely this: 

```
for chunk in client.send_message(bot, message, chatCode='1klcvn5ws9ne6en1f2z'):
    print(chunk)
    pass

```

Results in:
M
1
 
c
h
i
p
s
 
r
e
f
e
r
